data/saas/config/simon_data_config.yml

```diff
@@ -0,0 +1,42 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Simon Data
+  type: simon_data
+  description: A sample schema representing the Simondata connector for Fides
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      default_value: api.simondata.com
+    - name: api_token
+      label: API token
+      sensitive: True
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: bearer
+      configuration:
+        token: <api_token>
+
+  test_request:
+    method: GET
+    path: /v1/privacy/export
+
+  endpoints:
+    - name: contact
+      requests:
+        delete:
+          method: POST
+          path: /v1/privacy/delete/
+          body: |
+            {
+                "action": "delete",
+                "description": "Generated by Fides",
+                "email_address": "<email>",
```

Comment:
do we have docs on why `action` and `user_id_type` should be included here? I didn't see them listed as required here for email address -> https://documenter.getpostman.com/view/11533956/T17Q6jiE#11cf8a48-8d81-4b7e-b5ac-6bf143cdf388